modules/setup/setup.js

```diff
@@ -119,29 +155,37 @@ class Setup {
                         return;
                     }
                     progressbar.update(progress);
-                    }, 1000);
+                    }, SET_INTERVAL_TIME);
```

Comment:
A variable should have meaningful name which can help others to understand your code. SET_INTERVAL_TIME doesn't make any sense!!!